.flowconfig

```diff
@@ -0,0 +1,13 @@
+[ignore]
+.*/tests/.*
+.*node_modules/.*
+
+[include]
+src/
+
+[libs]
+interfaces/
+
+[options]
+suppress_comment= \\(.\\|\n\\)*\\$FlowFixMe
```

Comment:
Sometimes there is complication where you need to rewrite some of the code to guard against possibly nullable types. From reading the documentation, we don't actually need this one anymore as it's part of the defaults (must have been a change since I last read config docs heh)